### PR TITLE
Remove SQL defaults from `@comet/cms-api`

### PR DESCRIPTION
**Pull request by dkarnutsch**
This is in line with our best practices.

### CODE REVIEW

File: .changeset/fifty-mugs-walk.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@comet/cms-api": major
+---
+
+Redirect: Make property active required
+
+This change is only breaking if you rely on `RederictInput.active` being nullable.
```
Comment (johnnyomair) [inline .changeset/fifty-mugs-walk.md:7]:
```suggestion
This change is only breaking if you rely on `RedirectInput.active` being nullable.
```

**Review by johnnyomair — COMMENTED** [review]
Why did you decide to add default values in code in some instances, but not in all? Because they're defined in the GraphQL inputs? I believe you'll need to add more changesets then, because some may work directly with the entities (see fixtures.console.ts) in our projects

**Comment by dkarnutsch** [comment]
> Why did you decide to add default values in code in some instances, but not in all? Because they're defined in the GraphQL inputs? I believe you'll need to add more changesets then, because some may work directly with the entities (see fixtures.console.ts) in our projects

You are right – that was in consistent. I now provide default values in all entities for easier usage in apps: 73b8696f99d8ff382a4b34174f0ae680ee02d1a0

File: packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts [hunk 2]
```diff
@@ -37,10 +37,9 @@ export abstract class PageTreeNodeBaseCreateInput {
     @ValidateNested()
     attachedDocument: AttachedDocumentInput;
 
-    @Field({ nullable: true })
+    @Field({ defaultValue: false })
```
Comment (johnnyomair) [inline packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts:40]: Do we need a default value here as we now have a default value in code?

File: packages/api/cms-api/src/redirects/entities/redirect-entity.factory.ts [hunk 3]
```diff
@@ -60,9 +60,9 @@ export class RedirectEntityFactory {
             @Field({ nullable: true })
             comment?: string;
 
-            @Property({ default: true })
+            @Property()
             @Field()
-            active: boolean;
+            active: boolean = true;
```
Comment (johnnyomair) [inline packages/api/cms-api/src/redirects/entities/redirect-entity.factory.ts:65]: Here as well?

File: packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts (hunk 2 above)
Comment (dkarnutsch) [reply packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts:40]: yes, because otherwise this would be a breaking change

Comment (johnnyomair) [reply packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts:40]: You could leave it optional in the GraphQL schema as there's now a default value in code. Then it wouldn't be a change to the public API at all

**Comment by dkarnutsch** [comment]
@johnnyomair I've addressed your feedback, which made the change compatible, so I've changed the target to the main branch.

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**